alb/terraform-examples/nsxt-cloud-deployment/nsxt-cloud.tf

```diff
@@ -0,0 +1,124 @@
+terraform {
```

Comment:
Set required version for Terraform. e.g. 1.1.7

Reply:
done